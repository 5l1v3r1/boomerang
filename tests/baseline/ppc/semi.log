ERROR: attempted to decode outside any known segment 0x0x100104e0
ERROR: attempted to decode outside any known segment 0x0x10010444
after removing matching assigns (%pc).
in proc main adding addrExp r1{0} - 32 to local table
in proc main adding addrExp r1{0} - 4 to local table
in proc main adding addrExp r1{0} + 4 to local table
in proc main adding addrExp r1{0} - 24 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure main ###
in proc main adding addrExp r1{0} - 32 to local table
in proc main adding addrExp r1{0} - 4 to local table
in proc main adding addrExp r1{0} + 4 to local table
in proc main adding addrExp r1{0} - 24 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
in proc main adding addrExp r1{0} - 20 to local table
