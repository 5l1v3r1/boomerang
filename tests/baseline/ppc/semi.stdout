Boomerang alpha 0.3.2 19/Aug/2010
loading...
decoding entry point...
decoding anything undecoded...
finishing decode...
found 1 procs
decompiling...
 considering main
 decompiling main
global type analysis for main
igenerating code...
output written to ./tests/outputs/ppc/semi
completed in 0 secs.
